--- a/app/src/main/zstd/CMakeLists.txt
+++ b/app/src/main/zstd/CMakeLists.txt
@@ -7,7 +7,7 @@
 cmake_minimum_required(VERSION 3.5)
 include(CheckCCompilerFlag)
 project(zstd-jni C ASM)
 file(STRINGS "version" NATIVE_VERSION)
 find_package(JNI)
-include_directories("src/main/native" "src/main/native/common" "src/main/native/compress" "src/main/native/decompress" "src/main/native/dictBuilder" "src/main/native/legacy")
+include_directories("src/main/native" "src/main/native/common" "src/main/native/compress" "src/main/native/decompress" "src/main/native/dictBuilder" "src/main/native/legacy" "src/main/native/dhjni")
 if (JNI_FOUND)
     include_directories(${JNI_INCLUDE_DIRS})
 endif (JNI_FOUND)
